src/python_framework_v02/troute/nhd_network_utilities_v02.py

```diff
@@ -710,17 +710,15 @@ def build_data_assimilation_folder(data_assimilation_parameters,run_parameters,t
         usgs_files = [usgs_timeslices_folder.joinpath(f) for f in usgs_files]
     else:
         print("No Files Found for DA")
-        # TODO: Handles this with a real exception
-    max_fill_1min = data_assimilation_parameters.get("data_assimilation_interpolation_limit", 59)
-    qc_trehsh = data_assimilation_parameters.get("qc_threshold", 1)
+        # TODO: Handle this with a real exception
 
     usgs_df = nhd_io.get_usgs_from_time_slices_folder(
         data_assimilation_parameters["wrf_hydro_da_channel_ID_crosswalk_file"],
-        run_parameters['dt'],
         usgs_files,
-        qc_trehsh,
-        max_fill_1min,
-        t0,
+        data_assimilation_parameters.get("qc_threshold", 1),
+        data_assimilation_parameters.get("data_assimilation_interpolation_limit", 59),
+        run_parameters["dt"],
+        run_parameters["t0"],
```

Comment:
this is the t0 variable I suggest we hand to the lastobs_df_output function in nhd_io